assignment/health_centers.js

```diff
@@ -1,5 +1,5 @@
 // Preloaded healthcenter JSON
-var healthCenters = [
+const healthCenters = [

```

Comment:
We did not say it explicitly, but you shouldn't edit this file, it's just the data source